Remora.Discord.Hosting/Services/DiscordService.cs

```diff
@@ -69,6 +75,53 @@ ILogger<DiscordService> logger
     /// <inheritdoc />
     protected override async Task ExecuteAsync(CancellationToken stoppingToken)
     {
+        // do not check after the initial check at bot startup.
```

Comment:
This method is called once, so I find this comment to be moot?

Reply:
Well, once*, but even then changing host background service faulting behavior is niche in it of itself.

Reply:
I thought it could be called multiple times in the case of direct “return;" to loop back and recall it.

Reply:
That would both be non-trivial to determine, and rather asinine behavior. [See the docs](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.hosting.backgroundservice.executeasync)